src/client/Microsoft.Identity.Client/AuthScheme/PoP/PopBrokerAuthenticationScheme.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Cache.Items;
+using Microsoft.Identity.Client.Internal;
+
+namespace Microsoft.Identity.Client.AuthScheme.PoP
+{
+    //Authentication Scheme used when MSAL Broker and pop are used together.
+    internal class PopBrokerAuthenticationScheme : IAuthenticationScheme
+    {
+        public string AuthorizationHeaderPrefix => Constants.PoPAuthHeaderPrefix;
+
+        public string KeyId => string.Empty;
```

Comment:
Please add a comment that MSAL will not store tokens that come from brokers.

Reply:
And short tidbit about why (cached in broker, etc.)
Because potential question from users might be - so if MSAL doesn't cache these, does every request go to AAD via broker?

Reply:
In fact, keyId should not be used at all. How about we throw an InvalidOperationException?